src/pages/trade-types/margin/_start-trading.js

```diff
@@ -32,7 +33,7 @@ const StartTrading = () => {
                             )}
                         </Timeline.Item>
                     </Timeline>
-                    <LinkButton mt="4rem" to="/dmt5/" secondary="true">
+                    <LinkButton mt="4rem" to={deriv_mt5_demo_app_url} secondary="true">
```

Comment:
This won't work with the new configuration of LocalizedLink as it's an external link.

`<LinkButton mt="4rem" external="true" type="deriv_app" to="/mt5#demo" target="_blank" rel="noopener noreferrer" secondary="true">` should be the right use case. Is that right @yashim-deriv?

Reply:
He can use directly the `type="mt5"`

```
<LinkButton mt="4rem" external="true" type="mt5" to="#demo" target="_blank" rel="noopener noreferrer" secondary="true">
```

Reply:
followed latest LocalizedLink setup in the current commit.